api/openapi.yaml

```diff
@@ -0,0 +1,94 @@
+components:
+  schemas:
+    v1.Account:
+      properties:
+        account_number:
+          type: string
+        id:
+          maximum: 1.8446744073709552e+19
```

Comment:
I mean, we probably do not want to set any maximum/minimum value, both should be just strings I guess.

Reply:
Ah I understand now, this was generated by the tool and its because in our database AccountID is actually `uint64` for some reason. This should have been a string: https://github.com/RedHatInsights/sources-api-go/blob/main/model/tenant.go#L11